### ISSUE TITLE
Unable to start properly after installation on Windows

### ISSUE THREAD

**Issue by melston** [koka-lang/koka#512]
I just installed Koka on Windows.  I started with the VSCode extension, which then asked me to download and install the compiler.  After executing that I got a pop-up saying the 'Koka installation finished but unable to find the installed compiler."
I tried executing Koka from the command-line but got a long list of 'parse...' and 'check...' lines followed by a bunch of build errors:

```
std/core(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core.c")
std/core/delayed(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_delayed.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_delayed.c")
std/core/debug(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_debug.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_debug.c")
std/core/console(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_console.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_console.c")
std/core/show(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_show.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_show.c")
std/core/tuple(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_tuple.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_tuple.c")
std/core/either(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_either.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_either.c")
std/core/maybe(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_maybe.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_maybe.c")
std/core/list(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_list.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_list.c")
std/core/sslice(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_sslice.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_sslice.c")
std/core/string(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_string.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_string.c")
std/core/vector(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_vector.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_vector.c")
std/core/char(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_char.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_char.c")
std/core/order(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_order.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_order.c")
std/core/int(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_int.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_int.c")
std/core/bool(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_bool.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_bool.c")
std/core/exn(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_exn.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_exn.c")
std/core/hnd(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_hnd.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_hnd.c")
std/core/unsafe(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_unsafe.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_unsafe.c")
std/core/undiv(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_undiv.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_undiv.c")
std/core/types(1, 1): build error: user error (error  : does not exist
command: gcc -Wall -Wextra -Wpointer-arith -Wshadow -Wstrict-aliasing -Wno-unknown-pragmas -Wno-missing-field-initializers -Wno-unused-parameter -Wno-unused-variable -Wno-unused-value -Wno-unused-but-set-variable -g -Og -c -I c:/Users/mark.elston/AppData/Local/koka/share/koka/v3.1.1/kklib/include -DKK_MIMALLOC=8 -o "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_types.obj" "c:/Users/MARK~1.ELS/AppData/Local/Temp/.koka/v3.1.1/mingw-debug-5dd86a/std_core_types.c")
unable to load the std/core module; standard functions are not available
evaluation is disabled
```

Any ideas on what I did wrong or what to do to fix this?

**Comment by jameswoodleybetenbough** [comment]
Maybe similar to https://github.com/koka-lang/koka/issues/455.

**Comment by woojamon** [comment]
Maybe similar to https://github.com/koka-lang/koka/issues/455.

**Comment by melston** [comment]
> Maybe similar to #455.

Unfortunately, not.  I already have the VS Desktop C++ development workload installed (VS 2022).  I followed the trail from that issue and saw that I needed to run (something - koka itself?) in a Developers Command Prompt, so I tried that and got different errors.  Something about not being able to access the pdb file.  I have never had trouble using VS with C++ before.  I even tried doing this in a command prompt run as administrator and still no luck.

But, notice that the commands above use gcc instead of the VS compiler.  I don't know why that is, either (they were run from a powershell window).

**Comment by melston** [comment]
I have deleted the entire koka tree and I am trying the command-line install instead.  I find that even when using the developers command-line window I still get the following during installation:
```
Cannot find the clang-cl compiler.
A C compiler is required for Koka to function.
Would you like to download and install clang 17.0.6 for Windows? [Yn]
```
So, even if I have VC++ installed and in the environment, I still get asked about downloading clang compiler.  However, this time it seems to have worked.  I got to the interactive terminal just fine.
